src/Standards/Squiz/Sniffs/Operators/ComparisonOperatorUsageSniff.php

```diff
@@ -116,7 +116,16 @@ public function process(File $phpcsFile, $stackPtr)
                         if (isset($tokens[$i]['scope_closer']) === true) {
                             break;
                         }
-                    }
+                    } else if ($tokens[$i]['code'] === T_OPEN_PARENTHESIS) {
+                        // Stop if this is the start of a pair of
+                        // parentheses that surrounds the inline
+                        // IF statement.
+                        if (isset($tokens[$i]['parenthesis_opener']) === true
```

Comment:
Just checking: did you mean for this to be `'parenthesis_closer'` ?

Reply:
Hmm, as far as I understand (but I may be mistaken), `parenthesis_opener` and `parenthesis_closer` always occur in pairs, so both conditions would be equivalent. I meant this condition as "if this parenthesis opens a parenthesis scope and it's corresponding closer is beyond the inline if...". For me, the check on `parenthesis_opener` was a natural translation of the first part of that condition. If that makes the code more confusing, I'm happy to change it to `parenthesis_closer` however.

Reply:
Well, what parse errors or live coding ? The open parenthesis may exist, but the closer may not yet have been entered.

As you're comparing against the `parenthesis_closer` on the next line, using the same in the `isset()` should make the code more stable against potential future tokenizer changes and will prevent potential `undefined index` notices.

Reply:
Ah, in that case I've probably misunderstood how `parenthesis_opener` and `parenthesis_closer` work. I've changed the `isset` to check for `parenthesis_closer`. Thanks for pointing this out!